exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/internal/OtlpConfigUtil.java

```diff
@@ -237,9 +239,27 @@ public static void setMemoryModeOnOtlpMetricExporterBuilder(
                 "setMemoryMode", MemoryMode.class);
         method.setAccessible(true);
         method.invoke(builder, memoryMode);
+      } else if (builder instanceof OtlpGrpcSpanExporterBuilder) {
+        // Calling getDeclaredMethod causes all private methods to be read, which causes a
```

Comment:
I think this could have been avoided by using method handles. Alternatively could have used an internal interface to expose this method.